2018-08-02 02:06:53,631 ERROR 21319 nodejs.ClusterClientNoResponseError: client no response in 6481576ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21319
hostname: B000000126747

2018-08-02 03:33:35,370 ERROR 21319 nodejs.ClusterClientNoResponseError: client no response in 5181619ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21319
hostname: B000000126747

2018-08-02 05:21:52,505 ERROR 21319 nodejs.ClusterClientNoResponseError: client no response in 6477000ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21319
hostname: B000000126747

2018-08-02 07:09:57,512 ERROR 21319 nodejs.ClusterClientNoResponseError: client no response in 6484948ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21319
hostname: B000000126747

2018-08-02 08:58:02,501 ERROR 21319 nodejs.ClusterClientNoResponseError: client no response in 6484332ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21319
hostname: B000000126747

2018-08-02 09:09:22,528 ERROR 21319 nodejs.ClusterClientNoResponseError: client no response in 679508ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21319
hostname: B000000126747

2018-08-02 09:11:31,720 ERROR 21319 nodejs.ClusterClientNoResponseError: client no response in 88953ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21319
hostname: B000000126747

2018-08-02 09:25:05,649 ERROR 21319 nodejs.ClusterClientNoResponseError: client no response in 721186ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21319
hostname: B000000126747

2018-08-02 09:33:44,685 ERROR 21319 nodejs.ClusterClientNoResponseError: client no response in 518864ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21319
hostname: B000000126747

2018-08-02 11:34:22,052 ERROR 21319 nodejs.ClusterClientNoResponseError: client no response in 73764ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21319
hostname: B000000126747

2018-08-05 16:03:37,577 ERROR 13819 [-/127.0.0.1/-/7ms POST /assets/add] nodejs.TypeError: this.app.mysql.insert is not a function
    at AssetService.insert (/Users/sufubo/code/demo/asset-server/app/service/index.js:10:45)
    at HomeController.saveData (/Users/sufubo/code/demo/asset-server/app/controller/home.js:18:33)
    at Object.callFn (/Users/sufubo/code/demo/asset-server/node_modules/egg-core/lib/utils/index.js:36:21)
    at Object.classControllerMiddleware (/Users/sufubo/code/demo/asset-server/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/sufubo/code/demo/asset-server/node_modules/egg-core/lib/utils/index.js:36:21)
    at wrappedController (/Users/sufubo/code/demo/asset-server/node_modules/egg-core/lib/utils/router.js:322:18)
    at dispatch (/Users/sufubo/code/demo/asset-server/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/sufubo/code/demo/asset-server/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at /Users/sufubo/code/demo/asset-server/node_modules/koa-router/lib/router.js:346:16
    at dispatch (/Users/sufubo/code/demo/asset-server/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)

pid: 13819
hostname: B000000126747

2018-08-05 16:03:55,035 ERROR 13819 [-/127.0.0.1/-/1ms POST /assets/add] nodejs.TypeError: this.app.mysql.insert is not a function
    at AssetService.insert (/Users/sufubo/code/demo/asset-server/app/service/index.js:10:45)
    at HomeController.saveData (/Users/sufubo/code/demo/asset-server/app/controller/home.js:18:33)
    at Object.callFn (/Users/sufubo/code/demo/asset-server/node_modules/egg-core/lib/utils/index.js:36:21)
    at Object.classControllerMiddleware (/Users/sufubo/code/demo/asset-server/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/sufubo/code/demo/asset-server/node_modules/egg-core/lib/utils/index.js:36:21)
    at wrappedController (/Users/sufubo/code/demo/asset-server/node_modules/egg-core/lib/utils/router.js:322:18)
    at dispatch (/Users/sufubo/code/demo/asset-server/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/sufubo/code/demo/asset-server/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at /Users/sufubo/code/demo/asset-server/node_modules/koa-router/lib/router.js:346:16
    at dispatch (/Users/sufubo/code/demo/asset-server/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)

pid: 13819
hostname: B000000126747

2018-08-05 17:17:59,189 ERROR 15076 nodejs.ClusterClientNoResponseError: client no response in 1377247ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 15076
hostname: B000000126747

2018-08-05 19:06:53,589 ERROR 16519 nodejs.ClusterClientNoResponseError: client no response in 1858946ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 16519
hostname: B000000126747

2018-08-05 20:05:19,354 ERROR 16519 nodejs.ClusterClientNoResponseError: client no response in 264985ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 16519
hostname: B000000126747

2018-08-06 00:17:41,580 ERROR 16519 nodejs.ClusterClientNoResponseError: client no response in 919576ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 16519
hostname: B000000126747

2018-08-06 02:05:50,236 ERROR 16519 nodejs.ClusterClientNoResponseError: client no response in 6488249ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 16519
hostname: B000000126747

2018-08-06 05:41:59,930 ERROR 16519 nodejs.ClusterClientNoResponseError: client no response in 12949418ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 16519
hostname: B000000126747

2018-08-06 07:30:03,914 ERROR 16519 nodejs.ClusterClientNoResponseError: client no response in 6483802ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 16519
hostname: B000000126747

2018-08-06 09:18:08,696 ERROR 16519 nodejs.ClusterClientNoResponseError: client no response in 6484710ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 16519
hostname: B000000126747

2018-08-06 09:19:31,793 ERROR 16519 nodejs.ClusterClientNoResponseError: client no response in 63035ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 16519
hostname: B000000126747

2018-08-06 09:25:17,778 ERROR 16519 nodejs.ClusterClientNoResponseError: client no response in 254981ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 16519
hostname: B000000126747

2018-08-06 09:46:19,238 ERROR 16519 nodejs.ClusterClientNoResponseError: client no response in 1261290ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 16519
hostname: B000000126747

2018-08-06 09:50:09,176 ERROR 16519 nodejs.ClusterClientNoResponseError: client no response in 145143ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/sufubo/code/demo/asset-server/node_modules/cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 16519
hostname: B000000126747

